scripts/calculate-stats.ts

```diff
@@ -0,0 +1,76 @@
+import { globSync, readFileSync, writeFileSync } from 'node:fs';
+import { join } from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+import ts from 'typescript';
+
+function* iterate(node: ts.Node): Generator<ts.Node, void, unknown> {
+  yield node;
+  const childNodes: ts.Node[] = [];
+  ts.forEachChild(node, (n) => {
+    childNodes.push(n);
+  });
+  for (const childNode of childNodes) {
+    yield* iterate(childNode);
+  }
+}
+
+const dist = fileURLToPath(import.meta.resolve('../dist/'));
+const stats = {
+  size: 0,
+  cssSize: 0,
+  files: {} as Record<string, { size: number; cssSize?: number }>,
+};
+const themePath = 'elements/standard-theme.css';
```

Comment:
What about looking for files with "-theme.css" and also using a the named key (`standardCss`) below? It would be more future proof